src/app/canvastable/canvastable.ts

```diff
@@ -233,6 +233,8 @@ export class CanvasTableComponent implements AfterViewInit, DoCheck {
     }
   }
 
+  private dragSelectDirection: Boolean = null;
```

Comment:
I assume this is up/down; maybe it could be either commented or named to indicate what is what? Perhaps `dragSelectionIsDown` or something along those lines?

Reply:
Good point! Changed to `dragSelectionDirectionIsDown`.

Reply:
Excellent :)